frontend/packages/kubevirt-plugin/integration-tests/v2v/rhv/v2v.rhv.import.negative.scenario.incorrect.names.ts

```diff
@@ -0,0 +1,63 @@
+// import { browser, by } from 'protractor';
+// import { until } from 'selenium-webdriver';
+import {
+  asyncForEach,
+  click,
+  // waitForStringInElement
+} from '@console/shared/src/test-utils/utils';
+import { VirtualMachineModel } from '@console/kubevirt-plugin/src/models';
+// import {
+//   // CHARACTERS_NOT_ALLOWED_CAPITAL,
+//   SEC,
+// } from '../../tests/utils/constants/common';
+import { RhvImportWizard } from '../../tests/models/rhvImportWizard';
+import * as rhvView from '../../views/rhvImportWizard.view';
+import * as view from '../../views/importWizard.view';
+import { rhvVMConfigNoStartOnCreate } from './v2v.rhv.configs';
+// import {
+//   // asWrongDisk,
+//   asWrongNic,
+// } from '../../tests/utils/utils';
+import { wrongValues } from '../../tests/utils/constants/vm';
+// import { clickKebabAction } from '../../views/wizard.view';
+
+const { provider, instanceConfig, sourceVMName } = rhvVMConfigNoStartOnCreate;
+
+describe('VMWare Wizard validation, negative tests', () => {
+  const wizard = new RhvImportWizard();
+
+  beforeEach(async () => {
+    await wizard.openWizard(VirtualMachineModel);
+  });
+
+  afterEach(async () => {
+    await wizard.closeWizard();
+  });
+
+  async function processInstanceConnection() {
```

Comment:
can we do a similar thing what we did in https://github.com/openshift/console/pull/6525#discussion_r483609029 ? 
It prevents the code duplication and enhances reusability.

Reply:
I think I've splitt that logic as much as I can, processInstanceConnection differs for rhvImportWizard and vmwareImportWizard, so I will not be able to re-use it somewhere else. I think that is enough here.

Reply:
@suomiy I couldn't get what exactly was done there, please elaborate

Reply:
Done